src/main/java/geekbrains/java2/ht1b/Main.java

```diff
@@ -21,19 +21,28 @@
      */
 
     //глобальные переменные
-    static String[][] board = {{"A1", "A2", "A3"}, {"B1", "B2", "B3"}, {"C1", "C2", "C3"}};
-    static Scanner scanner = new Scanner(System.in);
+    private static String[][] board;
 
+    static {
+        board = new String[][]{{"A1", "A2", "A3"}, {"B1", "B2", "B3"}, {"C1", "C2", "C3"}};
+    }
+
+    private static Scanner scanner;
+
+    static {
+        scanner = new Scanner(System.in);
+    }
 
-    public static void main( String[] args ){
+
+    public static void main( String[] args ) {
         System.out.println("Игра в крестики-нолики");
         printBoard();
 
-        while(computerMove() != "Больше ходов нет. Ничья"){
+        while(computerMove() != "Больше ходов нет. Ничья") {
```

Comment:
\[Checkstyle\] ERROR: Literal Strings should be compared using equals\(\), not '\!='\.